cmd/opm/registry/add.go

```diff
@@ -78,6 +85,15 @@ func addFunc(cmd *cobra.Command, _ []string) error {
 	if err != nil {
 		return err
 	}
+	overwrite, err := cmd.Flags().GetBool("overwrite-latest")
+	if err != nil {
+		return err
+	}
+
+	enableAlpha, err := cmd.Flags().GetBool("enable-alpha")
+	if err != nil {
+		return err
+	}
```

Comment:
That shows fine for me. However, how can we ensure that the options will work well as well? 
Should we not add tests for those?